go.mod

```diff
@@ -1,12 +1,11 @@
-module github.com/c3os-io/c3os/provider-kubeadm
+module github.com/kairos-io/kairos/provider-kubeadm
 
 go 1.18
 
 require (
 	github.com/c3os-io/c3os/sdk v0.0.0-20220809204311-e169fab54d21
```

Comment:
Does this also need to change to kairos-io ?

Reply:
I didn't face any issues when building with this SDK. I will update this to kairos and try to build it.